.github/workflows/contracts.yml

```
@@ -0,0 +1,74 @@
+      
+      - name: Run Tests
+        working-directory: contracts/tip-escrow
+        run: cargo test --verbose
+      
+      - name: Build Contract
+        working-directory: contracts/tip-escrow
+        run: cargo build --target wasm32-unknown-unknown --release
+      
+      - name: Check WASM Size
+        working-directory: contracts/tip-escrow
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:3:6: Double quote to prevent globbing and word splitting

(shellcheck)